static/main.js

```diff
@@ -21,7 +21,7 @@ function init() {
     }
 
     var macros = {};
-    var options = {};
+    var options = {displayMode: true};
```

Comment:
Add a toggle for this.

Reply:
Sorry, I'm a little unclear. Is this a TODO for this PR? Or something longer term?

Reply:
I think it's okay to do it as a separate commit.  I'll open an issue once this lands.